internal/infra/updater.go

```diff
@@ -118,6 +126,28 @@ func NewUpdater(ctx context.Context, cli *client.Client, net *Networks, params *
 	return updater, nil
 }
 
+var ErrInvalidVolume = fmt.Errorf("invalid volume syntax")
+
+func mountOptions(v string) (local, remote string, readOnly bool, err error) {
```

Comment:
I tried but ParseMountRaw takes a source like `local` in `local:/remote` and seems to remove it since it's not an absolute path, I guess? So I'd have to parse it to add the working directory anyway making this not a useful thing to do.